include/oneapi/dpl/experimental/kt/esimd_radix_sort_onesweep.h

```diff
@@ -20,6 +20,79 @@
 
 namespace oneapi::dpl::experimental::esimd::impl
 {
+// float, 32bit
```

Comment:
Could you also leave a comment describing why using `std::numeric_limits<T>::max()` or `std::numeric_limits<T>::lowest()` as an identity does not work when performing radix sort of floating point numbers?